common/src/main/java/com/wynntils/services/mapdata/MapFeaturePoiWrapper.java

```diff
@@ -114,6 +114,7 @@ public void renderAt(
             float zoomRenderScale,
             float zoomLevel,
             boolean showLabels) {
+        // showLabels = fullScreenMap, to be removed when POIs are fully removed, and replaced by MapFeatureRenderer
```

Comment:
This only stays while we are porting. When we get rid of Pois and MapFeaturePoiWrapper, we can change showLabels to fullscreenMap officially.